README.md

```diff
@@ -36,9 +49,9 @@ The first (and currently only) implementation is based on spray-json. It support
 //
 // Normally, of course, you would only need one case class,
 // which would always represent the current version (V3 in this case).
-import stamina.json._
-import stamina.json.SprayJsonMacros._
+import fommil.sjs.FamilyFormats._
```

Comment:
+1 on the change.

In general, I think this import deserves a comment of some sorts or we should just define a simple spray-json format for the example class. The auto-generated json formats thing can be very confusing and it might distract from the point we are trying to make with the example.